README.md

```diff
@@ -1,31 +1,63 @@
-A helper library to assist in pulling in handlebars content both on the server and client side.  Provides a handful of helper functions (see ["functions"](#Functions) below).
+# Introduction
 
-# Server-side installation instructions
+This package provides components to assist in rendering handlebars templates both on the server and client side.
 
-This is currently an unlisted repository.  To add this library to your server-side dependencies, add it to your package.json file, as in:
+# Server-side Usage
+
+To add this library to your server-side dependencies, add it to your `package.json` file, as in:
 
 ```
   ...
   "dependencies": {
-    "fluid-handlebars-helper": "git://github.com/the-t-in-rtf/fluid-handlebars-helper.git",
+    "gpii-handlebars": "git://github.com/the-t-in-rtf/gpii-handlebars.git",
   },
   ...
 ```
 
-For examples of server-side usage, take a look at the tests in ```src/tests/js/server-tests.js```.
+## Handlebars Middleware
+
+The handlebars middleware provided in this package is a wrapper around [`express-handlebars`](https://github.com/ericf/express-handlebars).  Like that package, it provides a renderer that can be used with the Express `Response` object.  That renderer expects to start with a layout, and to replace the `{{body}}` variable in that layout with the contents of a template (`foo.handlebars` in our example).  Templates can references "partials", using notation like `{{>partial}}` or `{{>partial variable}}`.  These will be replaced with the context of the partial,
+rendered using either a supplied variable or the current context.
+
+To use this middleware, you need to make it aware of one or more directories that contain templates (typically via the `options.config.express.views` option in your `gpii.express` configuration.  Each of these view directories can have the following subdirectories:
+
+1. `layouts`
+2. `pages`
+3. `partials`
+
+The handlebars middleware supports inheritance between multiple view directories.  Layouts, pages and partials will be rendered from the first matching view directory.
+
+## Dispatcher Router
+
+The dispatcher router turns the last part of a path (such as `/dispatcher/foo`) into a template name (such as `foo`),
+and then attempts to find and render that template.  The dispatcher router is configured using the same options as the handlebars middleware, and supports the same kind of inheritance.
+
+## Inline Router
+
+The inline router reads all of the template content from one or more view directories and bundles this content up so that it can be used by the client-side renderer.  The inline router is configured using the same options as the handlebars middleware, and supports the same kind of inheritance.
 
-# Client-side installation instructions
+# Client-side Usage
 
-To make this available on the client side, add it to your bower dependencies, as in:
+To install this package on the client side, add it to your bower dependencies, as in:
```

Comment:
What makes you think bower is unmaintained?  They seem pretty active to me (they cut a new release last month, for example).